src/css/components/_tiles-only.scss

```
@@ -32,6 +32,14 @@
     text-align: center;
     text-decoration: none;
     position: relative;
+    @include flex-direction(column-reverse);
+    justify-content: space-evenly;
+}
+
+.tiles-only-template .soft-button-image {
+    height: auto;
+    max-width: 80%;
+    max-height: 60%;
 }
 
```

```suggestion

.tiles-only-template .soft-button p {
    padding: 0;
}
```

This can be added to make it look better. Also, it appears that the text overflows outside of the box, so that might need to be looked at as well

---

Added `padding: 0;` and `max-width: 80%;` in https://github.com/smartdevicelink/generic_hmi/pull/210/commits/8435f914b4ace356d458ca0bda593b2461c0ef6a so that text won't overflow.